### PR TITLE
Add Apple News icon

### LINKED ISSUE #7770: Request: Apple News
### Brand Name

Apple News

### Website

https://www.apple.com/apple-news/

### Popularity Metric

Apple News brings you the world’s best journalism, all in one place. Trusted sources, curated by editors, and personalized for you. An Apple News+ subscription makes it even better with access to hundreds of premium magazines and leading newspapers.

### Official Resources for Icon and Color

svg: https://upload.wikimedia.org/wikipedia/commons/7/75/Apple_News_2019_icon_%28iOS%29.svg  
color: `#FD415E` from Wikimedia 

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by shindgewongxj**
**Issue:** closes #7770 

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by dirien** [comment]
Hi @shindgewongxj,

thanks for the contribution. Can you open an issue, fill out the necessary informations and link to this PR?

**Comment by shindgewongxj** [comment]
> Hi @shindgewongxj,
> 
> thanks for the contribution. Can you open an issue, fill out the necessary informations and link to this PR?

OFC!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -918,6 +918,11 @@
             "hex": "FA243C",
             "source": "https://www.apple.com/itunes/marketing-on-music/identity-guidelines.html#apple-music-icon"
         },
+        {
+            "title": "Apple News",
+            "hex": "FD3159",
```
Comment (LitoMore) [inline _data/simple-icons.json:923]: The color should be `#F03355`. The color is from the subscribe button on https://www.apple.com/apple-news/.

Please keep your issue content updated to date.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json:923]: The icon resource is required. Please provide.

Comment (shindgewongxj) [reply _data/simple-icons.json:923]: > The color should be `#F03355`. The color is from the subscribe button on https://www.apple.com/apple-news/.
> 
> Please keep your issue content updated to date.

It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?

Comment (LitoMore) [reply _data/simple-icons.json:923]: > It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?

Where can I see the gradient values? Could you provide the resource that you picked from?

Comment (shindgewongxj) [reply _data/simple-icons.json:923]: > > It is linear gradient from `#FD3159` to `#FD5464`. For this situation, how should the regular work?
> 
> Where can I see the gradient values? Could you provide the resource that you picked from?

It seems I made a mistake, sorry...

**Comment by LitoMore** [comment]
CleanShot 2022-09-13 at 15 42 13@2x

I see some differences. Could you give a check?

**Comment by shindgewongxj** [comment]
> CleanShot 2022-09-13 at 15 42 13@2x
> 
> I see some differences. Could you give a check?

Screenshot 2022-09-13 at 4 56 24 PM
Is that the `Apple News` svg I committed, I reviewed the changes and it showed normally.

**Comment by LitoMore** [comment]
@shindgewongxj Please refer to this comment: https://github.com/simple-icons/simple-icons/pull/7752#issuecomment-1238557522.

**Comment by shindgewongxj** [comment]
> @shindgewongxj Please refer to this comment: [#7752 (comment)](https://github.com/simple-icons/simple-icons/pull/7752#issuecomment-1238557522).

I updated the commit with svgo precision 4.

**Comment by shindgewongxj** [comment]
> @shindgewongxj The linting error needs to be fixed.
> 
> You could open the SVG file in text mode and then delete the final-new-line.

I'm sorry, I cannot find any extraneous whitespace or line.

**Comment by LitoMore** [comment]
@shindgewongxj This is the final-new-line.

CleanShot 2022-09-13 at 17 54 22@2x

Fixed for you. Some editors will insert a final-new-line to the file automatically.

I will review the difference later.

**Comment by LitoMore** [comment]
CleanShot 2022-09-13 at 18 20 37@2x

@shindgewongxj Perfect fit now!

**Review by LitoMore — APPROVED** [review]
@shindgewongxj Thank you for your contributions!

**Pull request merged**